emission/storage/decorations/common_place_queries.py

```diff
@@ -0,0 +1,78 @@
+import emission.core.wrapper.common_place as ecwcp
+import emission.core.get_database as edb
+import emission.simulation.markov_model_counter as esmmc
+import emission.storage.decorations.common_trip_queries as esdctp
+
+#################################################################################
+############################ database functions #################################
+#################################################################################
+
+def save_common_place(common_place):
+    db = edb.get_common_place_db()
+    db.save(common_place)
+
+def get_common_place_from_db(_id):
+    db = edb.get_common_place_db()
+    json_obj = db.find_one({"_id" : _id})
+    return make_common_place_from_json(json_obj)
+
+def get_all_common_places_for_user(user_id):
+    db = edb.get_common_place_db()
+    return db.find({"user_id" : user_id})
+
+def make_common_place_from_json(json_obj):
+    return ecwcp.CommonPlace(json_obj)
+
+def make_new_common_place(user_id, coords, successors):
+    props = {
+        "user_id" : user_id,
+        "coords" : coords,
+        "common_place_id" : "%s%s" % (user_id, coords["coordinates"]),
+        "successors" : successors
+    }
+    return ecwcp.CommonPlace(props)
+
+def make_common_place(props):
```

Comment:
This is just an easy way to create a new trip without having to write a new props, 
the make_common_place_from_json is for when we get it out of mongo